src/routes/comments.py

```
@@ -0,0 +1,96 @@
+            )
+            db.add(notification)
+
+    await db.commit()
+    await db.refresh(new_comment)
+    return new_comment
+
+
+@router.post("/{comment_id}/like")
+async def like_comment(
+    comment_id: int,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)